internal/pkg/exec/docker.go

```diff
@@ -159,3 +172,57 @@ func imageName(uri, tag string) string {
 	}
 	return fmt.Sprintf("%s:%s", uri, tag)
 }
+
+// IsEcrCredentialHelperEnabled return true if ecr-login is enabled either globally or registry level
+func (c DockerCommand) IsEcrCredentialHelperEnabled(uri string) bool {
+	// Make sure the program is able to obtain the home directory
+	splits := strings.Split(uri, "/")
+	if c.homePath != "" && len(splits) > 0 {
+		// Look into the default locations
+		pathsToTry := []string{filepath.Join(".docker", "config.json"), ".dockercfg"}
+		for _, path := range pathsToTry {
+			content, err := ioutil.ReadFile(filepath.Join(c.homePath, path))
+			if err != nil {
+				// if we can't read the file keep going
+				continue
+			}
+
+			config, err := parseCredFromDockerConfig(content)
+			if err != nil {
+				continue
+			}
+
+			if config.CredsStore == CredStoreECRLogin || config.CredHelpers[splits[0]] == CredStoreECRLogin {
+				return true
+			}
+		}
+	}
+
+	return false
+}
+
+func parseCredFromDockerConfig(config []byte) (*dockerConfig, error) {
+	/*
+	Sample docker config file
+	"credsStore" : "ecr-login",
+	  "credHelpers": {
+	        "dummyaccountId.dkr.ecr.region.amazonaws.com": "ecr-login"
+	  },
```

Comment:
```suggestion
    {
        "credsStore" : "ecr-login",
        "credHelpers": {
            "dummyaccountId.dkr.ecr.region.amazonaws.com": "ecr-login"
        }
    }
```
Could you make this json file aligned?